package.json

```
@@ -9,32 +9,29 @@
   },
   "type": "module",
   "engines": {
-    "node": ">= 14"
+    "node": ">= 16"
   },
-  "alias": {
-    "app": "./app",
-    "api": "./app/api",
-    "assets": "./app/assets",
-    "behaviors": "./app/modules/general/behaviors",
-    "lib": "./app/lib",
-    "modules": "./app/modules",
-    "vendor": "./vendor"
+  "imports": {
+    "#app/*": "./app/*.js",
+    "#assets/*": "./app/assets/*.js",
+    "#behaviors/*": "./app/modules/general/behaviors/*.js",
```

since we are here, update svelte introduction

```suggestion
    "#behaviors/*": "./app/modules/general/behaviors/*.js",
    "#components/*": "./app/modules/general/components/*.js",
```